src/Collage/Render.elm

```diff
@@ -330,3 +342,56 @@ decodeDashing ds =
   ds
     |> List.map decodeOnOff
     |> String.join " "
+
+
+-- BASED ON https://github.com/rough-stuff/rough/blob/e9b0fdf36952a7a0f02e8015f4abac1ad39981c5/src/renderer.ts#L367
+bezierCurvePath : List Point -> String
+bezierCurvePath ps =
+    let
+        neighbors arr index =
+            Maybe.map4
+                (\m1 p1 i p2 ->
+                    ((m1, p1), (i, p2))
```

Comment:
This is a trick because Elm doesn't support tuples with more than 3 elements isn't it? Maybe use a record like `{prv, cur, nxt, sub}` for previous, current, next/sequent, subsequent? Or maybe `(prv, cur, (nxt, sub))` when using tuples?

Reply:
It was because the control points are calculated based on the slope between `m1` and `p1` and then `i` and p2`. But I think you're right that it would make more sense just to put them all in order here. Using a record would be a bit more readable but I need to use tuples to be able to destructure the x and y values in the case expression below.